test/sharness/t0062-daemon-api.sh

```diff
@@ -12,20 +12,21 @@ test_init_ipfs
 differentport=$((PORT_API + 1))
 api_different="/ip4/127.0.0.1/tcp/$differentport"
 api_unreachable="/ip4/127.0.0.1/tcp/1"
-api_fromcfg=$(ipfs config Addresses.API)
-peerid=$(ipfs config Identity.PeerID)
+
+test_expect_success "config setup" '
+	api_fromcfg=$(ipfs config Addresses.API) &&
+	peerid=$(ipfs config Identity.PeerID)
+'
 
 test_client() {
-	args="$@"
-	printf $peerid >expected
-	ipfs $args id -f="<id>" >actual
+	printf "$peerid" >expected &&
+	ipfs "$@" id -f="<id>" >actual &&
```

Comment:
ah, interesting. i didn't test it. i assumed from expectations about strings. ok